include/backup_restore.hpp

```diff
@@ -55,6 +55,14 @@ class BackupRestore
      */
     std::tuple<types::VPDMapVariant, types::VPDMapVariant> backupAndRestore();
 
+    /**
+     * @brief An API to update whether backup and restore functionality is being
```

Comment:
brief needs to be wrapped with in a line.

Reply:
Done